sllm/serve/controller.py

```diff
@@ -183,6 +184,57 @@ async def get_models(self):
         async with self.metadata_lock:
             return self.registered_models
 
+    async def status(self):
+        """
+        Returns the status of all registered models in OpenAI-compliant format.
+        """
+        async with self.metadata_lock:
+            models = []
+            for model_name, config in self.registered_models.items():
+                # Extract or calculate relevant fields
+                created_time = config.get("created", None)
+                max_model_len = config.get("max_position_embeddings", None)
+
+                model_permission_id = f"modelperm-{model_name}"
+                permission = [
```

Comment:
Do not return any non-existent information. It's okay to return `None`, but avoid providing misleading details.